resources/http_acl.rb

```diff
@@ -18,11 +18,93 @@
 # limitations under the License.
 #
 
-actions :create, :delete
-default_action :create
+include Chef::Mixin::ShellOut
+include Windows::Helper
 
-attribute :url, kind_of: String, name_attribute: true, required: true
-attribute :user, kind_of: String
-attribute :sddl, kind_of: String
+property :url, String, name_property: true, required: true
+property :user, String
+property :sddl, String
+property :exists, [true, false], desired_state: true
 
-attr_accessor :exists
+# See https://msdn.microsoft.com/en-us/library/windows/desktop/cc307236%28v=vs.85%29.aspx for netsh info
+
+load_current_value do |desired|
```

Comment:
Instead of loading the exists property we should just make this a helper method and use that method in a guard around a converge_by block. Creating properties that exist just to store state seems like a bad pattern to push out to community